configs/components/rubygem-ffi.rb

```diff
@@ -1,6 +1,6 @@
 component "rubygem-ffi" do |pkg, settings, platform|
-  pkg.version '1.9.25'
-  pkg.md5sum "e8923807b970643d9e356a65038769ac"
+  pkg.version '1.12.2'
```

Comment:
Most (all) runtime projects use ffi. Maybe it is not a good idea to bump it. We could ship a different version for bolt but that might not be worth the effort bc I think the failures we are seeing are due to https://github.com/puppetlabs/puppet-runtime/pull/294#issuecomment-582505408